runtime/onert/core/src/compiler/Compiler.cc

```diff
@@ -105,27 +105,31 @@ void ManualSchedulerOptions::setBackendMap(const ir::Model &model, const std::st
     const auto &key_str = key_val.at(0);
     const auto &val = key_val.at(1);
     auto key = static_cast<uint32_t>(std::stoi(key_str));
-
+#if 0
+    // Let setBackemdMap to parse str and put key, value pair to map.
+    // Check is done during execution.
     model.at(ir::SubgraphIndex{0})
       ->operations()
       .at(ir::OperationIndex{key}); // Check if exist, or this wil throw
+#endif
```

Comment:
It is the only place that uses `model` to check the operation index exists or not.
The same check is done in `ManualScheduler::schedule()`.

https://github.com/Samsung/ONE/blob/a0fbb99ee34ae112e97b2f1a5d2ba165f2f3908c/runtime/onert/core/src/compiler/ManualScheduler.cc#L90